db/types/operations/cast.py

```diff
@@ -64,11 +63,6 @@ def create_boolean_casts(engine):
     create_cast_functions(PostgresType.BOOLEAN, type_body_map, engine)
 
 
-def create_date_casts(engine):
-    type_body_map = _get_date_type_body_map()
-    create_cast_functions(PostgresType.DATE, type_body_map, engine)
-
-
```

Comment:
Since we create identical cast functions in `create_datetime_casts()`, I've removed this function.

Reply:
Nice catch.